src/modules/ie-objects/services/ie-objects.service.ts

```diff
@@ -275,6 +284,23 @@ export class IeObjectsService {
 		return allAdapted;
 	}
 
+	public async findObjectsForSitemap(licenses: IeObjectLicense[]): Promise<IeObjectsSitemap[]> {
```

Comment:
REQ: rename findObjectsForSitemap to **findIeObjectsForSitemap**. Everything is called IeObject and not just object for a reason.